README.md

```diff
@@ -70,9 +70,45 @@ Main features of diktat are the following:
 To **autofix** all code style violations use `-F` option.
 
 ## Run with Maven
+### Use the diktat-maven-plugin
+This plugin is available since version 0.1.3. You can see how it is configured in our project for self-checks: [pom.xml](pom.xml).
+If you use it and encounter any problems, feel free to open issues on [github](https://github.com/cqfn/diktat/issues).
```

Comment:
do think this is needed. More text in readme is worse

Reply:
It was already there, I just rearranged instructions for our plugin and antrun task

Reply:
As discussed, I'll only keep instructions on our plugins.